docs/en/enterprise-edition/rn/known-issues/known-fixed-issues.adoc

```diff
@@ -24,6 +24,24 @@ The list of fixed issues are not cumulative; only the issues that are fixed with
 //On *Inventory > Assets*, if you filter based on the _Key-Value_ *Asset Tag* and your environment has more that 1 million assets, the results will be inconclusive.
 //Contact your Prisma Cloud Customer Success representative for more details.
 
+|*RLP-133444*
+//Added post 24.2.2 when the issues was discovered. Moved to Fixed Issues in 3.1 preserving the details of the Known Issue.
+
+|The *Resource Tag Filter* option was incorrectly enabled for all users on the Prisma Cloud console. Ennablement is restricted to users who signed up for the Limited General Availability (LGA) trial.
```

Comment:
The Asset Tag filter was incorrectly added to the Asset Inventory for all users of the Prisma Cloud console. This filtering capability is available only for participants of a Limited General Availability program. 

@jenjoe22